tests/environment/docker-compose-aims.yml

```diff
@@ -109,11 +109,11 @@ services:
   alfresco-identity-service:
     image: quay.io/alfresco/alfresco-identity-service:${AIMS_TAG}
     environment:
-      - KEYCLOAK_USER=admin
-      - KEYCLOAK_PASSWORD=admin
-      - KEYCLOAK_IMPORT=/tmp/alfresco-realm.json
-      - DB_VENDOR=h2
+      - KEYCLOAK_ADMIN=admin
+      - KEYCLOAK_ADMIN_PASSWORD=admin
+      - KC_DB=dev-mem
```

Comment:
Out of curiosity, why did we need `dev-mem` here?

Reply:
I just wanted to replace the `h2` with something from the recent Keycloak version. I had `dev-file` and `dev-mem` to chose from. I selected `dev-mem` without any particular reason ;)